modules/network/vpc/variables.tf

```diff
@@ -118,44 +122,24 @@ variable "primary_subnetwork" {
   type        = map(string)
   default     = null
   validation {
-    condition     = var.primary_subnetwork == null || can(var.primary_subnetwork["subnet_name"])
-    error_message = "Primary subnetwork must define \"subnet_name\"."
-  }
-  validation {
-    condition     = var.primary_subnetwork == null || can(var.primary_subnetwork["subnet_region"])
-    error_message = "Primary subnetwork must define \"subnet_region\"."
-  }
-  validation {
-    condition     = var.primary_subnetwork == null || can(var.primary_subnetwork["new_bits"])
-    error_message = "Primary subnetwork must define \"new_bits\"."
+    condition     = var.primary_subnetwork == null
+    error_message = "primary_subnetwork is deprecated. Please see https://goo.gle/hpc-toolkit-vpc-deprecation for migration instructions."
   }
 }
 
 variable "additional_subnetworks" {
   description = "DEPRECATED: please see https://goo.gle/hpc-toolkit-vpc-deprecation for migration instructions"
   type        = list(map(string))
-  default     = []
-  validation {
-    condition = alltrue([
-      for s in var.additional_subnetworks : can(s["subnet_name"])
-    ])
-    error_message = "All additional subnetworks must define \"subnet_name\"."
-  }
-  validation {
-    condition = alltrue([
-      for s in var.additional_subnetworks : can(s["subnet_region"])
-    ])
-    error_message = "All additional subnetworks must define \"subnet_region\"."
-  }
+  default     = null
   validation {
-    condition     = alltrue([for s in var.additional_subnetworks : can(s["new_bits"])])
-    error_message = "All additional subnetworks must define \"new_bits\"."
+    condition     = var.additional_subnetworks == null
+    error_message = "additional_subnetworks is deprecated."
```

Comment:
Add `Please see https://goo.gle/hpc-toolkit-vpc-deprecation for migration instructions.`

Reply:
Good catch - I've added it now.